package.json

```diff
@@ -29,17 +29,17 @@
     "@types/semver": "^7.1.0",
     "@types/tar": "^6.0.0",
     "@types/which": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^6.8.0",
-    "@typescript-eslint/parser": "^6.8.0",
-    "@yarnpkg/eslint-config": "^1.0.0",
+    "@typescript-eslint/eslint-plugin": "^7.8.0",
+    "@typescript-eslint/parser": "^7.8.0",
+    "@yarnpkg/eslint-config": "^2.0.0",
     "@yarnpkg/fslib": "^3.0.0-rc.48",
     "@zkochan/cmd-shim": "^6.0.0",
     "babel-plugin-dynamic-import-node": "^2.3.3",
     "better-sqlite3": "^9.4.1",
     "clipanion": "^3.0.1",
     "debug": "^4.1.1",
     "esbuild": "0.19.5",
-    "eslint": "^8.0.0",
+    "eslint": "^8.57.0",
     "eslint-plugin-arca": "^0.16.0",
```

Comment:
```suggestion
```
The Yarn config already depends on this.

Reply:
Done!